mon-pix/app/utils/proposals-parser/text-block.js

```diff
@@ -0,0 +1,41 @@
+export default class TextBlock {
+
+  constructor({ text }) {
+
+    const firstCharacter = text.substring(0, 1);
+    const secondCharacter = text.substring(1, 2);
+    const isListItem = firstCharacter === '-'
+      || Number.isInteger(Number(firstCharacter)) && secondCharacter === '.';
```

Comment:
```suggestion
      || (Number.isInteger(Number(firstCharacter)) && secondCharacter === '.');
```
Ca fonctionne, mais je pense qu'on ne veut pas le cas  `-.toto`  ou alors il manque le cas de test.

Reply:
Pourquoi ce cas `-.toto` en particulier ?
`-.toto` serait un élément d'une liste suivant cet algo.

Reply:
Mb